nslsii/sync_experiment/sync_experiment.py

```diff
@@ -24,6 +24,17 @@ def get_current_cycle() -> str:
     return cycle_response.json()["cycle"]
 
 
+def get_commissioning_status(proposal_number, beamline) -> bool:
+    commissioning_proposals_response = nslsii_api_client.get(
```

Comment:
As this project continues to grow, it will be helpful to add docstrings, even to small functions.

```suggestion
def get_commissioning_status(proposal_number, beamline) -> bool:
    """True if proposal_number is registered as a commissioning proposal; else False."""
    commissioning_proposals_response = nslsii_api_client.get(
```